.gitignore

```diff
@@ -4,7 +4,6 @@
 .idea/
 .images/
 target/
-POLL/
```

Comment:
Please add `POLL/` to `.gitignore` again since testing poll command in development might create the directory and may publish personal data.

Reply:
Totally missed that. Changed it to /POLL/ so the "poll" folder in dao isn't ignored.